### ISSUE TITLE
Fix the bam after MergeBamAlignment

### ISSUE THREAD

**Issue by ge11232002** [uzh/ezRun#62]
MergeBamAlignment keeps the full sequence from uBam and adjusted the CIGAR.

The CIGAR is wrong when aligned to negative strand and shows mismatches in browser.

https://gatkforums.broadinstitute.org/gatk/discussion/11588/cigar-after-picard-mergebamalignment#latest

**Comment by ge11232002** [comment]
Reply from Picard forum:
```
Hi Ge,

I don't think we recommend doing any trimming. Have a look at this article for more information.

-Sheila
```

**Comment by ge11232002** [comment]
Fix via ab3ed6fe34103c4c1ae8917e6d2a2d7fd173cc26

**Comment by ge11232002** [comment]
New responses from Picard support people are not quite helping. Stick to current implementation.
